### DISCUSSION TITLE
Gas Gwei Not showing

### DISCUSSION THREAD

**Discussion by alan-joshy** [blockscout/blockscout#10557] [General]
i have enabled gas tracker still its not showing on frontend 

<img width="1496" alt="image" src="https://github.com/user-attachments/assets/ab16e9a9-b8ed-4aa7-86ab-da01b42133a3">

**Comment by ekawibisono** [comment]
Hello @alan-joshy Please Add ENV This. If you use docker-compose please add on file common-frontend.env
https://github.com/blockscout/frontend/blob/main/docs/ENVS.md#gas-tracker

**Reply by alan-joshy** [reply]
It was manual deployment and i have added these and still it was same need to try docker that might be easier

**Reply by ekawibisono** [reply]
> It was manual deployment and i have added these and still it was same need to try docker that might be easier

let's try use docker and i waiting your reply message
